xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-image/plugin.js

```diff
@@ -100,13 +101,283 @@
       }, this);
     },
     init: function(editor) {
+
+      // Register the the /img:: autocomplete
+      editor.config.mentions = editor.config.mentions || [];
```

Comment:
not sure to understand here, you mention img autocomplete in the comment but manipulate mentions just below.

Reply:
ah, unless 'mentions' is the name of the feature in CKEditor? In this case might be good to explain it in the comments as it might confuse somebody else :)

Reply:
I can see how this is confusing, the ckeditor mentions plugin is based on the autocomplete plugin, so technically it is also an autocomplete. I can change it to mention to avoid confusion